fhireval/data/__init__.py

```diff
@@ -0,0 +1,73 @@
+"""
+Test data is expected to be created to cover the needs for each of NCPI
+members' needs and will likely be aggregated into separate files. The data
+for each group can be stored as a single large json file or as separate 
+files. Each file will have some wrapper information that help sort the
+data so that test authors can access it easily enough.
+
+Each of these files must be valid json files with the following root level
+properties:
+  "groupname": 'Whatever the group name is, such as DbGAP or CMG"
+  "resources": { 
+      "resource_1" : [
+          list of valid fhir objects that can be "created" as part of a given
+          resource (so for patients, 'resource_1' above would be 'Patient'). 
+          These should all be of the same resource type
+      ],
+      "resource_2" : [
+          list of valid fhir objects, etc
+      ]
+}
+
+A special groupname, _ALL will be created containing a copy of all resources 
+found in each of the groups.
+
+It is assumed that the contents of each of the resource lists are valid and 
+should load uniquely into a fhir server
+"""
+import pkg_resources
+from pathlib import Path
+from collections import defaultdict
+import json
+
+# DO NOT import this. Just use the load_test_data function. It won't reload if the 
+# data has already been loaded. 
+_resource_data = None
+
+# Helper script to provide an easy way to clear out old test data
+def delete_content_for_resource(host, resource_type, records):
+    """host - will be a fixture 
+       resource_type - Patient, etc
+       records - one ore more fhir 'objects'. This function must identify the
+                 matches, get IDs and delete them"""

```

Comment:
Why don't we re-format docstrings against like: https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

Reply:
agreed. I have updated that one and will update any others as I encounter them